README.md

```diff
@@ -74,9 +74,9 @@ My lectures can be listened to as youtube tracks and lecture notes are also scan
   - [Practice Notes](https://github.com/kapelner/QC_Math_342W_Spring_2021/blob/master/practice_lectures/lec02.Rmd)
   - Theory Notes [(Rameasa Arna)](https://github.com/kapelner/QC_Math_342W_Spring_2021/blob/master/lectures/lec03arna.pdf) [(Jonathan Eng)](https://github.com/kapelner/QC_Math_342W_Spring_2021/blob/master/lectures/lec03eng.pdf) [(Steven Grgas)](https://github.com/kapelner/QC_Math_342W_Spring_2021/blob/master/lectures/lec03grgas.pdf)  -->
 * Lecture 2 video on slack
-  - Theory Notes [(Prof)](https://github.com/kapelner/QC_Math_342W_Spring_2021/blob/master/lectures/lec02kap.pdf)
+  - Theory Notes [(Prof)](https://github.com/kapelner/QC_Math_342W_Spring_2021/blob/master/lectures/lec02kap.pdf) [(Brendan Gubbins)](https://github.com/QC_MATH_342W_Spring_2021/blob/master/lectures/lec02gubbins.pdf)
 * Lecture 1 video on slack
-  - Theory Notes [(Prof)](https://github.com/kapelner/QC_Math_342W_Spring_2021/blob/master/lectures/lec01kap.pdf) [(Elizabeth McHugh)](https://github.com/QC_MATH_342W_Spring_2021/blob/master/lectures/lec01mchugh.pdf)
+  - Theory Notes [(Prof)](https://github.com/kapelner/QC_Math_342W_Spring_2021/blob/master/lectures/lec01kap.pdf) [(Elizabeth McHugh)](https://github.com/QC_MATH_342W_Spring_2021/blob/master/lectures/lec01mchugh.pdf) [(Brendan Gubbins)](https://github.com/QC_MATH_342W_Spring_2021/blob/master/lectures/lec01gubbins.pdf)
```

Comment:
You need to have "kapelner" there. This was my fault doing stuff late at night last night and I left it out of Elizabeth's pull request.

https://github.com/kaplener/QC_MATH_342W_Spring_2021/blob/master/lectures/lec01gubbins.pdf

Reply:
Thanks! Officially an open source contributor now